.prettierrc

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
None of our plugins have this, instead leveraging autofixing in eslint-config-fusion. I would prefer to isolate this change, and add this file everywhere if necessary, but I don't think it is?